canonical-web-and-design/vanilla-framework
# Add GitHub social icon

## Done

- adds GitHub icon to social icons
- driveby: fixes script to convert icons to mixins

Fixes #3276 

## QA

- Open [demo](https://vanilla-framework-3643.demos.haus/docs/patterns/icons#social)
- Check social icon examples, make sure GH icon renders correctly: 
  - https://vanilla-framework-3643.demos.haus/docs/examples/patterns/icons/icons-social
- Review updated documentation:
  - https://vanilla-framework-3643.demos.haus/docs/patterns/icons#social
  
### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="969" alt="Screenshot 2021-03-23 at 09 43 46" src="https://user-images.githubusercontent.com/83575/112118054-499bec80-8bbc-11eb-9d62-44dc69da4c37.png">
